service-discovery/README.adoc

```diff
@@ -31,304 +31,399 @@ WARNING: This content is generated by running npm --prefix .build run generate:a
+When the Red Hat OpenShift Application Services (RHOAS) Operator is installed in your OpenShift cluster, you can use the RHOAS CLI to interact directly with the cluster. For example, you can use the CLI to connect an application running in the cluster to a Kafka instance in {product-long}. Establishing a connection between an OpenShift application and a backing service such as {product} is called _service binding_.
+
+The following tutorial shows how to use the RHOAS CLI and RHOAS Operator to perform service binding. In the tutorial, you create an example Quarkus application and connect this to a Kafka instance. link:https://quarkus.io/[Quarkus^] is a Kubernetes-native Java framework that is optimized for serverless, cloud, and Kubernetes environments.
 
-The RHOAS Operator creates instances of {product} services in your cluster.
-Operator connects with {product} API giving you ability to embed connection details to various services.
+When you use the RHOAS CLI and RHOAS Operator for service binding, the RHOAS Operator injects connection credentials as files into the Pod for the application. Because the example Quarkus application uses the `quarkus-kubernetes-service-binding` link:https://quarkus.io/guides/deploying-to-kubernetes#service-binding[extension], the application automatically detects and uses the injected connection credentials.
 
-Operator currently enables developers to connect with their Red Hat OpenShift Streams for Apache Kafka (RHOSAK) instances.
-Operator is used by RHOAS CLI to connect your services with your OpenShift cluster
+In general, this automatic injection and detection of credentials eliminates the need to manually configure an application to connect to a Kafka instance in {product}. This is a particular advantage if you have numerous applications in your project that you want to connect to a Kafka instance.
 
-[.diagram]
-.Operator and CLI relationships
-image::rhoas-operator.png[Operator and CLI relationships]
+IMPORTANT: Some steps in the tutorial require cluster administrator access for your OpenShift cluster.
 
-NOTE: RHOAS Operator is already installed on various environments like OpenShift Developer Sandbox: https://developers.redhat.com/developer-sandbox
+=== Step 1: Verifying connection to your OpenShift cluster
 
-NOTE: Installation procedure assumes that you have access to install operators in openshift-namespace on your cluster.
+[role="_abstract"]
+In this step of the tutorial, you verify that the installed RHOAS Operator is working by using the RHOAS CLI to connect to your OpenShift cluster and retrieve the cluster status.
 
 .Prerequisites
-* You have login access to OpenShift cluster
-* You have cluster admin rights on OpenShift cluster in order to install Operator
+* The RHOAS Operator is installed in your OpenShift cluster. See xref:proc-installing-operator_{context}[Installing the RHOAS Operator in your OpenShift cluster].
+* You have cluster administrator access for your OpenShift cluster.
```

Comment:
I *think* on OpenShift Dedicated you don't get cluster admin except in CCS. There is another role you can get which is suitable for installing the operator If I Understand Correctly.

Reply:
Thanks, @pmuir. I think you mean the `dedicated-admin` role? https://docs.openshift.com/dedicated/4/administering_a_cluster/dedicated-admin-role.html

Reply:
yes, that sounds right - given OpenShift Dedicated is our key integration, calling that out specifically makes sense to me.